api/api/openapi.bundle.yaml

```diff
@@ -551,7 +551,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/inline_response_200'
+                $ref: '#/components/schemas/IdObject_1'
```

Comment:
This is automatically generated by the Makefile right?

Reply:
Yep that's right. I corrected the source openapi specs that was generated with `make bundle-openapi`